e2e/provision/playbooks/cluster.yml

```diff
@@ -24,6 +24,12 @@
       become: true
       when: (container_engine is not defined) or (container_engine == "docker")
       block:
+        - name: Add registry mirrors to docker configuration
+          ansible.builtin.set_fact:
+            docker_engine_daemon_json: 
+              registry-mirrors: "{{ lookup('ansible.builtin.env', 'DOCKER_REGISTRY_MIRRORS') }}"
+          when: lookup('ansible.builtin.env', 'DOCKER_REGISTRY_MIRRORS')
+
```

Comment:
@kispaljr correct me if I'm wrong, but it seems like `registry-mirrors` and `docker_engine_daemon_json` vars are not used anywhere. Do we need to set those facts?

Reply:
Nevermind, I just noticed that is for the [`docker_engine` role](https://github.com/andrewrothstein/ansible-docker_engine/blob/v0.2.2/tasks/main.yml#L34-L50)